ecs-init/config/common_test.go

```diff
@@ -82,3 +82,36 @@ func TestGetAgentPartitionBucketRegion(t *testing.T) {
 			})
 	}
 }
+
+const EnvCgroupMountpoint = "CGROUP_MOUNTPOINT"
+
+func TestCgroupMountpoint(t *testing.T) {
+	originalEnv := os.Getenv(EnvCgroupMountpoint)
+	defer os.Setenv(EnvCgroupMountpoint, originalEnv)
+
+	testcases := []struct {
+		name     string
+		env      string
+		expected string
+	}{
+		{
+			name: "system default",
+			env:  "", expected: cgroupMountpoint,
+		},
+		{
+			name: "from environment",
+			env:  "/other/mountpoint/cgroup", expected: "/other/mountpoint/cgroup",
```

Comment:
Nit: Unless it is a go convention, I'd rather `expected:` be on a separate line.

Reply:
Nope, I added the name and left the rest on the single line. Happy to add a new line for our eyes.